### PR TITLE
Move intro pages to dynamic intro

### PR DESCRIPTION
**Pull request by afjlambert**
The sttr-refactor added a dynamic-intro page for non-configured topics. This intro page can be used by our configured topics as well.
The main benefits are less duplicate code, better/more testing, more similar intropages (less divergence over time).

Please make sure:
- [x] to add a label
- [x] to add one or more reviewers
- [x] you followed our checklist for [functional testing](https://github.com/Amsterdam/vergunningcheck/blob/master/TESTING.md)
- [ ] ~~you added the necessary documentation (either in the markdown files or inline)~~ no need because the shared component is documented and CONTRIBUTING.md is still valid.
- [ ] ~~you added the necessary automated tests.~~ The intro-components are tested thoroughly. Not sure if testing this page (basically configuration) makes sense.

### CODE REVIEW

File: apps/client/src/intros/DakraamIntro.js [hunk 1]
```diff
@@ -1,42 +1,19 @@
...
-    <Paragraph>
-      <ContactSentence eventName={eventNames.INTRO_EXTRAORDINARY} />
-    </Paragraph>
-  </>
+  <Intro
+    usableForBullets={[
+      "het wijzigen van een monument.",
+      "het plaatsen van het dakraam zelf.",
+    ]}
+    usableForText={`Deze vergunningcheck gaat over dakramen, daklichten en lichtstraten. U
```
Comment (robinpiets) [inline apps/client/src/intros/DakraamIntro.js:11]: The render method for `usableForText` is not in this PR. That is already added in the `feature/sttr-refactor` branch. 

I don't understand if that's on purpose or not. Let's discuss, ok?

File: apps/client/src/intros/DakraamIntro.js [hunk 2]
```diff
@@ -1,42 +1,19 @@
...
+    kunt hem gebruiken als u een nieuwe plaatst of als u een bestaande
+    vernieuwt.`}
+    exceptions={[
```
Comment (robinpiets) [inline apps/client/src/intros/DakraamIntro.js:14]: Same for `exceptions`

**Review by robinpiets — CHANGES_REQUESTED** [review]

File: apps/client/src/intros/DakraamIntro.js (hunk 1 above)
Comment (afjlambert) [reply apps/client/src/intros/DakraamIntro.js:11]: It's on purpose. The Intro component is introduced by sttr-refactor. Now this intro page can use that same component.

File: apps/client/src/intros/DakraamIntro.js (hunk 2 above)
Comment (afjlambert) [reply apps/client/src/intros/DakraamIntro.js:14]: It's on purpose. The Intro component is introduced by sttr-refactor. Now this intro page can use that same component.

**Comment by svenjens** [comment]
I think the general intropages can be deleted once we start using translation strings

**Review by robinpiets — APPROVED** [review]
Only lots of merge conflicts

**Review by svenjens — APPROVED** [review]

**Comment by svenjens** [comment]
This can be merged right? @afjlambert

**Pull request merged**